locales/en/moves.json

```diff
@@ -146,5 +146,18 @@
   },
   "agreement_status": {
     "heading": "Has this move been agreed with receiving prison?"
+  },
+  "proposed_moves_flow": {
```

Comment:
These could be nested within the object we have for `dashboard` already. Could create an object with that for `filter`. That way we could share that if we use it across other views. Stops it being tied specifically to this _flow_:

```json
...
"dashboard": {
  ...
  "filter": {
    "proposed": "proposed",
    "approved": "approved",
    "rejected": "rejected"
  }
  ...
}
...
```

Reply:
Don't think we need the label property, that was more for form fields where we'd have multiple properties, one for label, one for hint, etc.

Also, I think they should be lowercased, like the GOV.UK Notify banner as they should read like a sentence: `0 proposed` instead of `0 Proposed`.